recipes/drfp/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "drfp" %}
+{% set version = "0.2.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/drfp-{{ version }}.tar.gz
+  sha256: ea4755ad989bbd31a62cc628a7ae1bacdf2bfef0dfde5c899cb4551917405640
+
+build:
+  number: 0
+  entry_points:
```

This looks like it's pure Python so this can be noarch Python:
```suggestion
  noarch: python
  entry_points:
```